root/static/scripts/edit/components/ArtistCreditEditor.js

```diff
@@ -103,14 +103,18 @@ class ArtistCreditEditor extends React.Component {
 
     const ac = this.state.artistCredit;
     const newState = mutate(this.state, newState => {
-      newState.artistCredit.names.splice(i, 1);
+      newState.artistCredit.names
+        .splice(i, 1);
```

Comment:
This looked fine/better how it was (if the rule complained, we should fix the rule).